feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.html

```diff
@@ -24,12 +24,12 @@
   >
     <option
       *ngFor="let item of attribute.values"
-      [label]="getLabel(expMode, item.valueDisplay, item.valueCode)"
       [selected]="item.selected"
+      [label]="getLabel(expMode, item.valueDisplay, item.valueCode, item)"
       [attr.aria-label]="getAriaLabel(item, attribute)"
       [value]="item.valueCode"
     >
-      {{ getLabel(expMode, item.valueDisplay, item.valueCode) }}
+      {{ getLabel(expMode, item.valueDisplay, item.valueCode, item) }}
```

Comment:
Plz, check with our a11y expert whether value are read correctly with and without price